code/modules/mob/living/carbon/human/species/station/uplifts.dm

```diff
@@ -112,7 +112,7 @@
 	damage_mask = 'icons/mob/human_races/masks/dam_mask_corvid.dmi'
 	blood_mask = 'icons/mob/human_races/masks/blood_corvid.dmi'
 
-	slowdown = -1
+	slowdown = -0.5
```

Comment:
Partial numbers are meaningless as a slowdown. -1 is the lowest possible buff.

Reply:
fixed